components/update_client/protocol_serializer_json.cc

```diff
@@ -70,11 +70,15 @@ std::string ProtocolSerializerJSON::Serialize(
   // OS version and platform information.
   base::Value::Dict os_node;
   os_node.Set("platform", request.os.platform);
-  os_node.Set("arch", request.os.arch);
-  if (!request.os.version.empty())
-    os_node.Set("version", request.os.version);
-  if (!request.os.service_pack.empty())
-    os_node.Set("sp", request.os.service_pack);
+  // TODO(b/456632915): Device info OS returns garbled characters.
```

Comment:
go/todo-format
Our TODOs should have the form `// TODO: b/123, Cobalt - Description`